Automattic/wp-calypso
# BBE: Do not set theme in flow

#### Proposed Changes

In the BBE flow (`/start/do-it-for-me`), the user can select the desired theme on the design picker step and once the flow completes, we set this theme on the site. Since premium themes can only be set on sites with a valid subscription, we excluded premium themes from the design picker. However, a Premium theme is required for BBE, and users will have access to premium themes anyway after checkout. More context: p1651811253362459/1651773699.112809-slack-C02KVCAL7GX

This PR skips setting the theme on the site so that we can eventually add Premium themes to the design picker.

#### Testing Instructions



* Go to `/start/do-it-for-me` , select "New Site", and go through the flow steps. Confirm that you can complete the purchase.
* Go to `/start/do-it-for-me` , select "Existing Site", and go through the flow steps. Confirm that you can complete the purchase.

#### Pre-merge Checklist





Fixes Automattic/martech#929
